src/components/App/App.js

```diff
@@ -5,16 +5,12 @@ import {
   Redirect,
   Switch,
 } from 'react-router-dom';
-
 import { connect } from 'react-redux';
-
 import Nav from '../Nav/Nav';
 import Footer from '../Footer/Footer';
-
 import ProtectedRoute from '../ProtectedRoute/ProtectedRoute';
-
 import AboutPage from '../AboutPage/AboutPage';
-import UserPage from '../UserPage/UserPage';
+import UserPage, { UserPageFuncExport } from '../UserPage/UserPage';
```

Comment:
This is what I was talking about yesterday as far as importing multiple components from the same file. 
Your UserPage is the default export. Therefore it can be imported directly. 
My component is not a default export. Instead it lives in an export object. So the brackets destructure the object on import allowing me to use the other component as well.